src/Files/UserControls/NavigationToolbar.xaml

```diff
@@ -562,6 +562,28 @@
                         Value="{x:Bind OngoingTasksViewModel.InfoBadgeValue, Mode=OneWay}" />
                 </Grid>
 
+                <Button
+                    x:Name="UpdateButton"
+                    x:Load="{x:Bind ViewModel.UpdateSettingsService.IsUpdateAvailable, Mode=OneWay}"
+                    Width="36"
+                    Height="32"
+                    AccessKey="U"
+                    AutomationProperties.AutomationId="UpdateButton"
```

Comment:
Is this needed?

Reply:
The width and height? Yeah, otherwise the button shrinks to the content size, and it looks a bit weird. I could move the height/width to the ToolBarButtonStyle. The Settings button has the same height/width.

Reply:
I meant the AutomationId

Reply:
Oh, no it's not needed.